invoice/views.py

```diff
@@ -38,7 +38,7 @@ def form_valid(self, form):
         """Create a new customer and a new address."""
         address_form = AddressForm(self.request.POST)
         if not address_form.is_valid():
-            return self.form_invalid(address_form)
+            return self.form_invalid(form)
```

Comment:
@iTitus was this the bug you mentioned yesterday with deleting everything?

Reply:
Yes, it was only showing the sub-form